invisible_cities/io/dst_io_test.py

```diff
@@ -188,3 +191,35 @@ def test_store_pandas_as_tables_unordered_df(config_tmpdir, df1, df2):
         store_pandas_as_tables(h5out, df2, group_name, table_name)
     df_read = load_dst(filename, group_name, table_name)
     assert_dataframes_equal(df_read, pd.concat([df1, df2], sort=True).reset_index(drop=True))
+
+
```

Comment:
Another good test could be opening the file and checking that the table is actually indexed:

`a_table.indexed` is a property that indicates whether a table is indexed
`a_table.cols.<col_name>.indexed` is a property that indicates whether a specific column is indexed.

Reply:
But this is just labeling columns to be indexed, is not indexing them (thats whats done at the end of the city for example using  `index_tables`function of components)
We could index table  in store_pandas_as_tables but in case it is called repeatedly inside cities it doesn't seem efficient.

Reply:
We have io/indexation_test.py that creates a dummy city and ensures every writer indexes the tables it writes. This should be added there. However this writer has a slightly different structure than the other, so it may require some adaptation.

We also need another PR to index tables in the cities that use this writer.